cryptographic_flows.md

```diff
@@ -164,6 +245,8 @@ For now, simple clear-text storage is acceptable.
 
 #### Server-side storage
 - [TODO #28](https://github.com/boltlabs-inc/key-mgmt-spec/issues/28). Include appropriate requirements for server-side secure storage and generate relevant issues in key-mgmt.
+##### Audit Logs
+The server storage should include a per-user audit log that tracks system registration and logins, key use requests, and audit log requests. A single log entry contains the following information about each action: action, actor, date, outcome (e.g. success or failure), any related key identifier.
```

Comment:
Is this the right place to discuss requirements around log rotation and backup for per-user audit logs? 

This is largely a deployment issue but would like to figure out the right balance between the availability of logs and preventing DoS on server storage.